src/Sylius/Bundle/ApiBundle/DependencyInjection/Compiler/SyliusLegacyAliasesPass.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class SyliusLegacyAliasesPass implements CompilerPassInterface
+{
+    public const PLUGIN_TO_SYLIUS_PREFIXES_MAP = [
+        'Sylius\PriceHistoryPlugin\Application\Validator' => 'Sylius\Bundle\ApiBundle\Validator',
+        'Sylius\PriceHistoryPlugin\Infrastructure\Serializer' => 'Sylius\Bundle\ApiBundle\Serializer',
```

Comment:
Using it like this creates way more aliases than necessary since it aliases every service within the `Sylius\Bundle\ApiBundle\Validator|Serializer` directories.
I'd be for being more specific and using one-to-one aliases instead.